ansible/playbooks/AdoptOpenJDK_AIX_Playbook/README.md

```diff
@@ -0,0 +1,61 @@
+# Setting up AIX servers
+
+-- Work in Progress -- (WIP)
+
+1) AIX BASE Requirements
+1a) X11 requirements
+# OJDK requires X11 support
+# Additional dependencies are installed by default
+X11.adt.ext
+# X11 Frame Buffers
+X11.vfb
+X11.base.lib
+
+1b) AIX NLS/language support
+# Highly recommended are the following multi-language support files
+# https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/657#issuecomment-454308017
+bos.iconv
+bos.loc.com.CN
+bos.loc.com.JP
+bos.loc.com.utf
+bos.loc.utf.EN_ZA
+bos.loc.iso.en_ZA
+bos.loc.iso.ja_JP
+bos.loc.iso.ko_KR
+bos.loc.iso.zh_CN
+bos.loc.pc
+bos.loc.utf.JA_JP
+bos.loc.utf.KO_KR
+bos.loc.utf.ZH_CN
+bos.loc.utf.ZH_TW
+
+1c) OPENSSL requirements
+The most recent versions of openssh and openssl, in any case
+openssl.base.1.0.2.1600 and later
+
+
+1d) AIX base requirements to be able to use AIX Toolbox
+OSS software packaged by AIX Toolbox will require the following filesets:
+expect
+tk
+tcl
+
+Additionally, the yum setup script (yum.sh) - if used, will require the ftp client. For AIX 7.2 that is a seperate fileset
+bos.net.tcp.ftp
+
+2) ## WIP ## Ansible playbook instructions
+
+3) (build without jenkins-agent)
```

Comment:
I don't think this should be in an AIX specific file. We need this to be generic across all platforms otherwise if we have the information in multiple places it's a lot more work to update if required. My PR at https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/2041/files adds this to one of the top level files

Reply:
I can cut it later - because for now I am working from the generic steps - and working on what is different when it is AIX.
I am thinking ease of use - for someone used to AIX but not Eclipse AdoptOpenJava.

For now I would like to work from premise:
a) include link to generic instructions
b) include additional information that is AIX platform specific

I do not have all the answers right now - which is why I consider it WIP.

Once 90%+ of the pieces are here - we can review what needs to be (re)moved to the generic instructions and what can stay here as platform specific.

Further, this approach is not new - I actually took ./AdoptOpenJDK_Windows_Playbook/README.md as my template for getting started.